quality-estimation/modelfront.md

```diff
@@ -746,11 +747,22 @@ supported_languages:
   base_code: zu
   name: Zulu
   variant_name: null
-only_compatible_mt_api: null
-only_compatible_tms: null
+only_compatible_mt_api:
```

Comment:
Seems easier for the layout code if this is just `only_compatible_mt_api: null` like before.